playground/backend/internal/validators/validator_test.go

```diff
@@ -0,0 +1,41 @@
+package validators
+
+import (
+	"fmt"
+	"os"
+	"testing"
+)
+
+func TestMain(m *testing.M) {
```

Comment:
I see that `java_validators_test.go` depends on this `validator_test.go` file since `CheckIsUnitTestJava` reads file that is created here. How exactly are these files connected?

Reply:
java_validators_test and validator_test - in the same package, they use the same TestMainmethod before running tests

Reply:
Got it, thanks!